patches/minecraft/net/minecraft/item/Food.java.patch

```diff
@@ -0,0 +1,73 @@
+--- a/net/minecraft/item/Food.java
++++ b/net/minecraft/item/Food.java
+@@ -11,15 +11,26 @@
+    private final boolean field_221472_c;
+    private final boolean field_221473_d;
+    private final boolean field_221474_e;
+-   private final List<Pair<EffectInstance, Float>> field_221475_f;
++   private final List<Pair<java.util.function.Supplier<EffectInstance>, Float>> field_221475_f;
++   
++   private Food(Food.Builder builder) {
++	   this.field_221470_a = builder.field_221458_a;
++	   this.field_221471_b = builder.field_221459_b;
++	   this.field_221472_c = builder.field_221460_c;
++	   this.field_221473_d = builder.field_221461_d;
++	   this.field_221474_e = builder.field_221462_e;
++	   this.field_221475_f = builder.field_221463_f;
++   }
```

Comment:
This is a obvious mix of tabs vs spaces, see later in the file.
And the patch could be so much smaller if this was the normal method redirect by having the old function call the new.